src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,17 @@
+$gap-calendar: 1px;
+$width-block-calendar-day: 100px;
+$days-in-row: 7;
+
+$count-of-days: 31;
+
+$icons: "mon" 7, "tue" 6,"wed" 5,"thu" 4,"fri" 3,"sat" 2,"sun" 1;
+$width-container-all: $width-block-calendar-day * $days-in-row + $gap-calendar * $days-in-row;
+
+
+$color-regular: #060b35;
+$color--button--blue: #00acdc;
+$color--main--white: #fff;
+
+$card-width: 200px;
+$star-size: 16px;
+$height-button: 40px;
```

Comment:
completely reduntant variables from previous task))

Reply:
@OlggaJul ha-ha 
did not notice) thanks